src/webchat-ui/components/WebchatUI.tsx

```diff
@@ -289,11 +290,17 @@ export class WebchatUI extends React.PureComponent<React.HTMLProps<HTMLDivElemen
     }
 
     sendMessage: MessageSender = (...args) => {
+        this.props.onSendMessage(...args);
+
         if (this.history.current) {
-            this.history.current.scrollToBottom();
+            /**
+             * this timeout is necessary to avoid
+             * an immediate "typing indicator" to not be scrolled correctly
+             */
+            setTimeout(() => {
+                this.history.current.scrollToBottom();
+            }, 150);
```

Comment:
Is this still necessary with a latest changes?

Reply:
fixed